src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -126,7 +126,7 @@ parseFixityDeclaration = do
   name <- symbol
   return $ FixityDeclaration fixity name alias
   where
-  aliased = (Left <$> parseQualified (Ident <$> identifier))
+  aliased = P.try (Left <$> parseQualified (Ident <$> identifier))
         <|> (Right <$> parseQualified (ProperName <$> uname))
```

Comment:
Also `ProperName <$> uname` is the same as `properName` from `Common` - should I switch it to that?